### ISSUE TITLE
Feature Request: AutoSneak/Drag

### ISSUE THREAD

**Issue by HanryuDR** [joemcbride/outlander-osx#99]
Auto sneak while hidden
Auto drag $dragtarget

I suppose this would involve intercepting "go ____" and all directionals (n,s,e,w,aft,starboard,etc). At least that's the way YASSE used to handle it. You could circumvent the auto-sneak/drag by typing "northea" or "g gate" as it wasn't looking for the longform of the cardinals or the short form of GO. Climb was the hard one. You can't "sneak ladder" you have to "climb ladder" but you "drag hanryu branch"

**Comment by joemcbride** [comment]
I think this would be a good feature to add, though I'm thinking of a simpler form.

With Genie you could add "mini-scripts" to macros.

```
#if {$hidden = 1}{#send sneak down}{#send down}
```

So that would be one way to handle it.

The other easier option is to add the sneaking/dragging to the automapper.  With just a few tweaks to the `automapper.cmd` script this could be accomplished today.  The existing genie automapper script does this I just haven't converted it yet.

**Comment by HanryuDR** [comment]
> #if {$hidden = 1}{#send sneak down}{#send down}

That is what I was trying to do and it wasn't working :)
The other "issue" is that 90% of the time I play from a laptop and thanks to apple's ongoing crusade to make things dumber, the fn key no longer enables a keypad on the keyboard so I'm typing e,e,n,w,s,e,e,etc a bunch. I guess I could turn those into aliases?

**Comment by joemcbride** [comment]
Yeah that "mini-scripts" isn't a feature I've added yet (should have clarified that).

Going the alias route would probably be painful.  You could do `ctrl-n`, `ctrl-e`, etc. as macros.  Not sure if that would be much better.  I could see wanting to "toggle-on" a set of macros just for navigation then toggling them back off.  So you could maybe use `QWASDE` as directions when that's toggled on.
